pkgs/development/python-modules/etesync/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, buildPythonPackage, fetchPypi,
+  appdirs, asn1crypto, cffi, cryptography, furl, idna, orderedmultidict,
+  packaging, peewee, py, pyasn1, pycparser, pyparsing, pyscrypt,
+  python-dateutil, pytz, requests, six, vobject,
+  pytest
+}:
+
+buildPythonPackage rec {
+  pname = "etesync";
+  version = "0.9.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "03016nwn67b0nk8gl6ycnz5c7b8jnxgyrypx9zv5sfara706xmd8";
+  };
+
+  propagatedBuildInputs = [
+    appdirs
+    asn1crypto
+    cffi
+    cryptography
+    furl
+    idna
+    orderedmultidict
+    packaging
+    peewee
+    py
+    pyasn1
+    pycparser
+    pyparsing
+    pyscrypt
+    python-dateutil
+    pytz
+    requests
+    six
+    vobject
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
+  checkPhase = ''
+    pytest tests/test_collections.py
+    pytest tests/test_crypto.py
+  '';
+}
```

Comment:
please add meta section

https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md

Reply:
whoops. done, sorry about that.

I feel kind of bad causing human roundtrips on stuff like this :hurtrealbad: but also man there is a lot of stuff to remember. Can't stuff like this be encoded in CI roundrips and admonished by bots? Anyways, thanks for your work!